src/parenttext_pipeline/configs.py

```diff
@@ -232,13 +232,29 @@ class MediaAssetSourceConfig(SourceConfig):
     storage: MediaStorage
     """Information about the media storage system to download assets from."""
 
+    server_storage: MediaStorage
+    """Information about the media storage system to upload assets to for RapidPro to access."""
```

Comment:
I think this is unnecessary since `MediaAssetDeploymentConfig` has been introduced.